frontend/packages/operator-lifecycle-manager/src/components/descriptors/spec/configure-update-strategy.tsx

```diff
@@ -13,7 +14,8 @@ export const configureUpdateStrategyModal = ({
     resourceKind: kindObj,
     resource,
     defaultValue: specValue,
-    title: `Edit ${specDescriptor.displayName}`,
+    // TODO Internationalize updateStrategyModela to accect titleKey and titleVariables.
```

Comment:
```suggestion
    // TODO Internationalize updateStrategyModal to accept titleKey and titleVariables.
```

I think the pattern here is OK. We use it in other places, so I'm not sure the TODO is needed, as long as this is a function that's called during render.

Reply:
I suggested this because it's a pattern we use in other modals. I wasn't sure if we were trying to avoid using the global i18n instance like this.

Reply:
I'll remove this comment. We can address this if it becomes an issue.